### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by radoslavkalinov** [Provenance-Emu/Provenance#1902]
Summary;

Since the previous build (the first version I installed) n64 emulation for most games I played the audio crashes at some point during gameplay and a constant motor like hum follows. St the time of writing this (version 2.1.0) this is still the case. Im running ios 15.2 on ipad pro 12.9 (2020)

 
N64 games to run without audio crashing

 
Audio crashes

 ### Reproduce:  

1. Start n64 game
2. Run for a while
3. Et voilá

# 
#### NOTES  

replaceWithNotesOrDelete

#

#### iOS/tvOS VERSION

-------------------------------------------

**Comment by klaus-richter** [comment]
Same issue on an ipad 10.2 8gen  
Thanks

**Comment by JoeMatt** [comment]
I've seen something similar but not as severe.
If you open the pause menu, and then continue, does the audio get better?

**Comment by klaus-richter** [comment]
@JoeMatt yes, but after 3 minutes the audio come back to crashes and so on

**Comment by JoeMatt** [comment]
> @JoeMatt yes, but after 3 minutes the audio come back to crashes and so on

ok thanks, that's helpful. probably a sync issue that resets when the pause state is activated and then starts to drift again.

**Comment by DanEEStar** [comment]
This problem happens also on my installation with the N64 but also with the SNES and the GBA emulator.
Interestingly not with PS1 and some others.

The sound works for up to about 2 minutes and then it starts to scramble. After opening the pause menu and then continue it works again for the next few minutes.

It started to happen without the new installation of the application but I think with an upgrade to a new iOS version (maybe from 14 to 15).

It just started to happen on my iPad Pro 10.5 and the only thing that changed in the mean time was an upgrade to iOS... 

The interesting thing IMO is, that also on the Delta emulator the same problem started to happen on my iPad also without a new installation of the Delta app: https://github.com/rileytestut/Delta/issues/188

**Comment by j0xaxr** [comment]
> This problem happens also on my installation with the N64 but also with the SNES and the GBA emulator. Interestingly not with PS1 and some others.
> 
> The sound works for up to about 2 minutes and then it starts to scramble. After opening the pause menu and then continue it works again for the next few minutes.
> 
> It started to happen without the new installation of the application but I think with an upgrade to a new iOS version (maybe from 14 to 15).
> 
> It just started to happen on my iPad Pro 10.5 and the only thing that changed in the mean time was an upgrade to iOS...
> 
> The interesting thing IMO is, that also on the Delta emulator the same problem started to happen on my iPad also without a new installation of the Delta app: [rileytestut/Delta#188](https://github.com/rileytestut/Delta/issues/188)

It can be a problem with iPadOS itself 
This problem happens on my iPad but not on my iPhone
And i have seen people who have this issue are mostly iPad users

**Comment by JoeMatt** [comment]
stale
